src/ace/managers/ptplayer.c

```diff
@@ -1215,7 +1216,7 @@ static void mt_playvoice(
 		pChannelData->n_reallength = uwSampleLength;
 
 		// Determine period table from fine-tune parameter
-		UBYTE ubFineTune = pSampleDef->ubFineTune; // d3
+		UBYTE ubFineTune = pSampleDef->ubFineTune & 0xF;
```

Comment:
This worries me a bit. Is it for some kind of malformed MODs or perhaps there are values above 15 which need some kind of different handling for topmost bits? A comment in the source code would be helpful here.

Reply:
I've checked all the MOD docs I know and haven't found any info about upper nibble except that it must be zero. I think the best would be having some kind of sanitizer for that value, but that's work for separate issue. A todo comment here should suffice.

```suggestion
		// TODO: sanitize this value to only have lower nibble set when loading MOD/samplepack
		UBYTE ubFineTune = pSampleDef->ubFineTune & 0xF;
```